maplibre/src/tile/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod tile_parser;
```

Comment:
I would favor to name this module `data` instead of `tile`. That way it is more generic and is compatible with the naming in the paper.